README.rst

```diff
@@ -31,9 +31,10 @@ Usage
     import schedule
     import time
 
+    """ Basic scheduling """
```

Comment:
```suggestion
```
I don't think we need to divide the readme into basic- and advanced sections. Everything is quite simple :)